examples/offline_inference.py

```diff
@@ -10,13 +12,33 @@
 # Create a sampling params object.
 sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
 
-# Create an LLM.
-llm = LLM(model="facebook/opt-125m")
-# Generate texts from the prompts. The output is a list of RequestOutput objects
-# that contain the prompt, generated text, and other information.
-outputs = llm.generate(prompts, sampling_params)
-# Print the outputs.
-for output in outputs:
-    prompt = output.prompt
-    generated_text = output.outputs[0].text
-    print(f"Prompt: {prompt!r}, Generated text: {generated_text!r}")
+# Use model from www.modelscope.cn
+# FROM_MODELSCOPE=True python examples/offline_inference.py \
+# --model="damo/nlp_gpt2_text-generation_english-base" \
+# --revision="v1.0.0" --from_modelscope
+if __name__ == "__main__":
```

Comment:
for the example, I would like to keep it simple. The documentation is sufficient.

Reply:
reverted